aiidalab_widgets_base/viewers.py

```diff
@@ -249,8 +250,8 @@ def nglview_parameters(self, indices):
         }
         if self.type.value == "ball+stick":
             nglview_parameters_dict["params"]["aspectRatio"] = self.size.value
-        else:
-            nglview_parameters_dict["params"]["radiusScale"] = 0.1 * self.size.value
+        elif self.type.value == "spacefill":
```

Comment:
Maybe add an else for not support type value and raise exception?

Reply:
I won't be able to test it, since `self.type` is a Dropdown that doesn't have another choice.

Reply:
Fair. `else` is not always necessary :)